ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift

```diff
@@ -0,0 +1,148 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveUI
+import Lottie
+import Preferences
+import SwiftUI
+
+struct FocusSystemSettingsView: View {
+  @Environment(\.colorScheme) private var colorScheme
+  @Environment(\.verticalSizeClass) var verticalSizeClass: UserInterfaceSizeClass?
+  @Environment(\.horizontalSizeClass) var horizontalSizeClass: UserInterfaceSizeClass?
+
+  @Binding var shouldDismiss: Bool
+
+  var shouldUseExtendedDesign: Bool {
+    return horizontalSizeClass == .regular && verticalSizeClass == .regular
+  }
+
+  let dynamicTypeRange = (...DynamicTypeSize.xLarge)
+
+  var body: some View {
+    NavigationView {
```

Comment:
Remove, its already in one

Reply:
All unnecessary navigation views are gone https://github.com/brave/brave-core/commit/91c6c912a8207d7f813f771f7e0e16ff5050eb1d